src/hooks/useWalletBalances.ts

```diff
@@ -3,19 +3,77 @@ import { useMemo } from 'react';
 import { Address } from 'viem';
 import useAccountSettings from './useAccountSettings';
 import { useAddysSummary } from '@/resources/summary/summary';
+import { useQueries } from '@tanstack/react-query';
+import { fetchPositions, positionsQueryKey } from '@/resources/defi/PositionsQuery';
+import { RainbowPositions } from '@/resources/defi/types';
+import { add, convertAmountToNativeDisplay } from '@/helpers/utilities';
+import { queryClient } from '@/react-query';
 
-const useWalletBalances = (wallets: AllRainbowWallets) => {
+type WalletBalance = {
+  assetBalanceAmount: number;
+  assetBalanceDisplay: string;
+  positionsBalanceAmount: string | number;
+  positionsBalanceDisplay: string;
+  totalBalanceAmount: string | number;
```

Comment:
this will always be a string as it is a result of `add` which is always a string